File: Models/ContactUs.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Models
{
    public class ContactUs
    {
        [Required(ErrorMessage = "Please provide valid name to contact you.")]
        public string Name { get; set; }
        [Required]
        [DataType(DataType.EmailAddress, ErrorMessage = "E-mail is not valid")]
        public string Email { get; set; }
        [Required(ErrorMessage = "Please provide message with in 50 words."), MaxLength(100)]
        public string Message { get; set; }
    }
}

File: CoreWebApp/ClientApp/src/app/Models/ContactUs.ts
export class ContactUs {
  /*[Required(ErrorMessage = "Please provide valid name to contact you.")]*/
  public Name: string;
  /*[Required]
          [DataType(DataType.EmailAddress, ErrorMessage = "E-mail is not valid")]*/
  public Email: string;
  /*[Required(ErrorMessage = "Please provide message with in 50 words."), MaxLength(100)]*/
  public Message: string;
}
